letsencrypt/tests/auth_handler_test.py

```diff
@@ -148,6 +148,30 @@ def test_perform_failure(self):
         self.assertRaises(
             errors.AuthorizationError, self.handler.get_authorizations, ["0"])
 
+    @mock.patch("acme.challenges.HTTP01Response.simple_verify")
+    def test_simple_verify(self, mock_verify):
+        from acme import jose
+
+        from letsencrypt.tests import test_util
+
+        domain = b'localhost'
+        key = jose.JWK.load(test_util.load_vector('rsa512_key.pem'))
+        http_01 = achallenges.KeyAuthorizationAnnotatedChallenge(
+            challb=acme_util.HTTP01_P, domain=domain, account_key=key)
+        tls_sni_01 = achallenges.KeyAuthorizationAnnotatedChallenge(
+            challb=acme_util.TLSSNI01_P, domain=domain, account_key=key)
+
+        dv_c = [http_01, tls_sni_01]
+
+        self.handler.dv_c = dv_c
+
+        self.mock_dv_auth.perform.side_effect = gen_http01_resp
+
+        mock_verify.return_value = False
+        # with mock.patch("letsencrypt.plugins.manual.logger") as mock_logger:
```

Comment:
don't try to check if logger was called enough times. check instead that `simp_le` verify was called enough times AND with correct arguments